src/core_modules/capture-core/components/WidgetStagesAndEvents/Stages/Stage/StageDetail/hooks/helpers.js

```diff
@@ -50,7 +54,7 @@ const convertStatusForView = (event: ApiTEIEvent) => {
     return {
         isNegative,
         isPositive,
-        text: translatedStatusTypes(options)[status],
+        text: translatedStatusTypes(options)[status] ?? event.status,
```

Comment:
I think we can safely remove the `?? event.status`? Also, I see that we now have two files called statusTypes (possibly due to some merge conflicts). Can we use the file in the events folder instead and delete the one from the DataElements folder?

Reply:
thanks @JoakimSM. I wanted to add `?? event.status` in case we have any unhandled statuses but if we are sure then it's not necessary. will remove it

Reply:
thanks @JoakimSM. I wanted to add `?? event.status` in case we have any unhandled statuses but if we are sure then it's not necessary. will remove it